azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/ArtifactHandlerImplV2.java

```diff
@@ -235,6 +253,40 @@ private String getResourceConfiguration() {
         return resources.stream().map(resource -> resource.toString()).collect(Collectors.joining(","));
     }
 
+    protected void deployExternalResources(DeployTarget deployTarget) throws AzureExecutionException {
+        final PublishingProfile publishingProfile = deployTarget.getPublishingProfile();
+        final String serverUrl = publishingProfile.ftpUrl().split("/", 2)[0];
+        try {
+            FTPClient ftpClient = FTPUtils.getFTPClient(serverUrl, publishingProfile.ftpUsername(), publishingProfile.ftpPassword());
+            List<Resource> externalResources = this.externalResources;
+            for (Resource resource : externalResources) {
+                uploadResource(resource, ftpClient);
+            }
+        } catch (IOException e) {
+            throw new AzureExecutionException(e.getMessage(), e);
+        }
+    }
+
+    protected void uploadResource(Resource resource, FTPClient ftpClient) throws IOException {
+        final List<File> files = Utils.getArtifacts(resource);
+        final String target = getAbsoluteTargetPath(resource.getTargetPath());
+        for (File file : files) {
+            FTPUtils.uploadFile(ftpClient, file.getPath(), target, log);
+        }
+    }
+
+    protected static String getAbsoluteTargetPath(String targetPath) {
+        // convert null to empty string
+        targetPath = StringUtils.defaultString(targetPath);
+        return StringUtils.startsWith(targetPath, "/") ? targetPath :
```

Comment:
use startwith '/' to determine whether path is a absolute path, refers https://github.com/frohoff/jdk8u-jdk/blob/master/src/solaris/classes/sun/nio/fs/UnixPath.java#L369